client/galaxy/scripts/apps/analysis.js

```diff
@@ -119,6 +122,13 @@ window.app = function app(options, bootstrapped) {
             this.page.display(new HistoryList.View({ action_id: action_id }));
         },
 
+        show_history_citations: function() {
+            var citationInstance = Vue.extend(Citations);
+            var vm = document.createElement("div");
+            this.page.display(vm);
+            new citationInstance({ propsData: { id: QueryStringParsing.get("id"), source: "histories" } }).$mount(vm);
```

Comment:
this spacing seems to be different from our standard

Reply:
It is whatever client-format did.

Reply:
I thought we abandoned extra spaces with regards to parens.